config/routes.rb

```diff
@@ -41,5 +41,6 @@
 
   namespace :merchant do
     get '/', to: 'dashboard#index', as: 'dashboard'
+    get '/items', to: 'items#index', as: 'user_items'
```

Comment:
Wasn't sure what the best name for this path should be. Thoughts?

Reply:
The way you set it up makes a lot of sense to me!